From: Denis Oliver Kropp <dok@shizo.zion.home>
Date: Tue, 9 Oct 2007 01:46:06 +0000 (+0200)
Subject: Made argument to fusion_skirmish_lock_count() const.
X-Git-Url: http://git.directfb.org/?p=core%2FDirectFB.git;a=commitdiff_plain;h=05cca1e20a8f14738b5b97a23afb8e15a2a54473

Made argument to fusion_skirmish_lock_count() const.

Added unimplemented fusion_skirmish_lock_count() for single app.
---

diff --git a/lib/fusion/lock.c b/lib/fusion/lock.c
index fbdedd3..9e1dc56 100644
--- a/lib/fusion/lock.c
+++ b/lib/fusion/lock.c
@@ -135,7 +135,7 @@ fusion_skirmish_swoop( FusionSkirmish *skirmish )
 }
 
 DirectResult
-fusion_skirmish_lock_count( const FusionSkirmish *skirmish, int *lock_count )
+fusion_skirmish_lock_count( FusionSkirmish *skirmish, int *lock_count )
 {
      int data[2];
 
@@ -372,7 +372,7 @@ fusion_skirmish_swoop( FusionSkirmish *skirmish )
 }
 
 DirectResult
-fusion_skirmish_lock_count( const FusionSkirmish *skirmish, int *lock_count )
+fusion_skirmish_lock_count( FusionSkirmish *skirmish, int *lock_count )
 {
      D_ASSERT( skirmish != NULL );
      
@@ -503,18 +503,6 @@ fusion_skirmish_swoop (FusionSkirmish *skirmish)
 }
 
 DirectResult
-fusion_skirmish_lock_count( const FusionSkirmish *skirmish, int *lock_count )
-{
-     D_ASSERT( skirmish != NULL );
-     
-     D_UNIMPLEMENTED();
-
-     *lock_count = 0;
-
-     return DFB_UNIMPLEMENTED;
-}
-
-DirectResult
 fusion_skirmish_dismiss (FusionSkirmish *skirmish)
 {
      D_ASSERT( skirmish != NULL );
diff --git a/lib/fusion/lock.h b/lib/fusion/lock.h
index 9740ffc..5d55e34 100644
--- a/lib/fusion/lock.h
+++ b/lib/fusion/lock.h
@@ -79,7 +79,7 @@ DirectResult fusion_skirmish_swoop  ( FusionSkirmish    *skirmish );
 /*
  * Find out how many times current thread has acquired lock. 
  */
-DirectResult fusion_skirmish_lock_count( const FusionSkirmish *skirmish, int *lock_count );
+DirectResult fusion_skirmish_lock_count( FusionSkirmish *skirmish, int *lock_count );
 
 /*
  * Unlock.
